Source/Scene/Scene.js

```diff
@@ -2070,6 +2085,24 @@ define([
         }
     }
 
+    function executeIdCommand(command, scene, context, passState) {
+        if (!defined(command.derivedCommands)) {
+            return;
+        }
+
+        if (scene._logDepthBuffer && defined(command.derivedCommands.logDepth)) {
```

Comment:
Minor, but why not `var derivedCommands = command.derivedCommands` to remove a lot of redundancy below?